test-network/network.sh

```diff
@@ -268,7 +268,6 @@ function networkUp() {
   # generate artifacts if they don't exist
   if [ ! -d "organizations/peerOrganizations" ]; then
     createOrgs
-    createConsortium
```

Comment:
Corresponding createConsortium function should also be removed (note, the check for configtxgen tool not found should be moved to the new first invocation of configtxgen).